src/controllers/lostDoc/updateItem.js

```diff
@@ -0,0 +1,37 @@
+import UpdateItem from '../../models/lost';
+
+export default async (req, res) => {
+        const {
+            documentName,
+            documentNumber,
+            isRewarded,
+            price
+        } = req.body;
+ try{
+    const edittedItem = await UpdateItem.findById(req.params._id);
```

Comment:
Can we find a different variable name for this, it might cause confusion as this is not the edited item rather the item that you want to edit.

Reply:
Sure! we can as to avoid confusion.